<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeepGcfg</class>
 <widget class="QWidget" name="DeepGcfg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,1,1,4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3,1">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Config Dir</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ConfigLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ConfigButton">
       <property name="text">
        <string>Open...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,3,1">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Network File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="NetworkLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="NetworkButton">
       <property name="text">
        <string>Import...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1,2,1,2,1">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Number of Input:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="InputNumSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Number of Params:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="ParamNumSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Dataset:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="DatasetComboBox">
       <property name="currentText">
        <string>mnist</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>mnist</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>fashion</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>cifar10</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="RunDeepG">
       <property name="text">
        <string>Run DeepG</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="ERANButton">
       <property name="text">
        <string>Run ERAN</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="runDeepsymbolButton">
       <property name="text">
        <string>Run DeepSymbol</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
